--- origsrc/gnome-pilot-2.32.1/conduits/backup/Makefile.am	2011-05-25 14:51:00.612476600 -0500
+++ src/gnome-pilot-2.32.1/conduits/backup/Makefile.am	2011-05-25 13:58:56.000000000 -0500
@@ -11,7 +11,7 @@ backup_conduits_LTLIBRARIES = \
 
 libbackup_conduit_la_SOURCES = backup_conduit.c backup_conduit.h 
 
-libbackup_conduit_la_LDFLAGS = -module -avoid-version
+libbackup_conduit_la_LDFLAGS = -module -avoid-version -no-undefined
 libbackup_conduit_la_LIBADD = 	\
 	$(GNOME_PILOT_LIBS) \
 	$(top_builddir)/gpilotd/libgpilotdconduit.la
--- origsrc/gnome-pilot-2.32.1/conduits/evolution-data-server/Makefile.am	2011-05-25 14:51:00.617476900 -0500
+++ src/gnome-pilot-2.32.1/conduits/evolution-data-server/Makefile.am	2011-05-25 13:58:56.000000000 -0500
@@ -26,7 +26,7 @@ libeaddress_conduit_la_SOURCES = \
 	e-pilot-util.h
 
 libeaddress_conduit_la_LDFLAGS = \
-	-module -avoid-version
+	-module -avoid-version -no-undefined
 
 libeaddress_conduit_la_LIBADD = \
 	$(GNOME_PILOT_LIBS) \
@@ -44,7 +44,7 @@ libecalendar_conduit_la_SOURCES = \
 	e-pilot-util.h
 
 libecalendar_conduit_la_LDFLAGS = \
-	-module -avoid-version
+	-module -avoid-version -no-undefined
 
 libecalendar_conduit_la_LIBADD = \
 	$(LIBECAL_LIBS) \
@@ -62,7 +62,7 @@ libememo_conduit_la_SOURCES = \
 	e-pilot-util.h
 
 libememo_conduit_la_LDFLAGS = \
-	-module -avoid-version
+	-module -avoid-version -no-undefined
 
 libememo_conduit_la_LIBADD = \
 	$(LIBECAL_LIBS) \
@@ -80,7 +80,7 @@ libetodo_conduit_la_SOURCES = \
 	e-pilot-util.h
 
 libetodo_conduit_la_LDFLAGS = \
-	-module -avoid-version
+	-module -avoid-version -no-undefined
 
 libetodo_conduit_la_LIBADD = \
 	$(GNOME_PILOT_LIBS) \
--- origsrc/gnome-pilot-2.32.1/conduits/file/Makefile.am	2011-05-25 14:51:00.623477300 -0500
+++ src/gnome-pilot-2.32.1/conduits/file/Makefile.am	2011-05-25 13:58:56.000000000 -0500
@@ -14,7 +14,7 @@ file_conduits_LTLIBRARIES = \
 
 libfile_conduit_la_SOURCES = file_conduit.c file_conduit.h palm.keys.in
 
-libfile_conduit_la_LDFLAGS = -module -avoid-version
+libfile_conduit_la_LDFLAGS = -module -avoid-version -no-undefined
 libfile_conduit_la_LIBADD = 		\
 	$(GNOME_PILOT_LIBS)  \
 	$(top_builddir)/gpilotd/libgpilotdconduit.la
--- origsrc/gnome-pilot-2.32.1/conduits/test/Makefile.am	2011-05-25 14:51:00.630477700 -0500
+++ src/gnome-pilot-2.32.1/conduits/test/Makefile.am	2011-05-25 13:58:56.000000000 -0500
@@ -11,7 +11,7 @@ test_conduits_LTLIBRARIES = \
 libtest_conduit_la_SOURCES = 	\
 	test_conduit.c
 
-libtest_conduit_la_LDFLAGS = -module -avoid-version
+libtest_conduit_la_LDFLAGS = -module -avoid-version -no-undefined
 libtest_conduit_la_LIBADD = 	\
 	$(GNOME_PILOT_LIBS) \
 	$(top_builddir)/gpilotd/libgpilotdconduit.la
--- origsrc/gnome-pilot-2.32.1/gpilotd/Makefile.am	2011-05-25 14:51:00.634477900 -0500
+++ src/gnome-pilot-2.32.1/gpilotd/Makefile.am	2011-05-25 14:32:23.734594900 -0500
@@ -132,11 +132,12 @@ libgpilotdcminclude_HEADERS = \
 	../libgpilotdCM/gnome-pilot-conduit-config.h
 
 libgpilotdcm_la_LIBADD =		\
+	libgpilotd.la			\
 	$(GNOME_PILOT_LIBS)		\
 	$(NULL)
 
 libgpilotdcm_la_LDFLAGS = \
-	-version-info $(GPILOTD_CM_CURRENT):$(GPILOTD_CM_REVISION):$(GPILOTD_CM_AGE)
+	-version-info $(GPILOTD_CM_CURRENT):$(GPILOTD_CM_REVISION):$(GPILOTD_CM_AGE) -no-undefined
 
 ## LIBGPILOTD
 
@@ -154,7 +155,7 @@ libgpilotd_la_LIBADD = \
 	$(NONE)
 
 libgpilotd_la_LDFLAGS = \
-	-version-info $(GPILOTD_CURRENT):$(GPILOTD_REVISION):$(GPILOTD_AGE)
+	-version-info $(GPILOTD_CURRENT):$(GPILOTD_REVISION):$(GPILOTD_AGE) -no-undefined
 
 ## LIBGPILOTDCONDUIT
 libgpilotdconduit_la_SOURCES = 			\
@@ -184,11 +185,12 @@ libgpilotdconduitinclude_HEADERS = 		\
 libgpilotdconduitincludedir = $(includedir)/gpilotd
 
 libgpilotdconduit_la_LIBADD =	\
+	libgpilotd.la		\
 	$(GNOME_PILOT_LIBS)	\
 	$(NULL)
 
 libgpilotdconduit_la_LDFLAGS = \
-	-version-info $(GPILOTD_CONDUIT_CURRENT):$(GPILOTD_CONDUIT_REVISION):$(GPILOTD_CONDUIT_AGE)
+	-version-info $(GPILOTD_CONDUIT_CURRENT):$(GPILOTD_CONDUIT_REVISION):$(GPILOTD_CONDUIT_AGE) -no-undefined
 
 ###
 
