--- origsrc/gnome-pilot-2.32.0/conduits/backup/backup.conduit.in	2009-08-06 13:59:19.000000000 -0500
+++ src/gnome-pilot-2.32.0/conduits/backup/backup.conduit.in	2010-12-14 19:25:30.394091000 -0600
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="gpbackup1" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/libbackup_conduit.so"/>
+	<conduit id="gpbackup1" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/cygbackup_conduit.dll"/>
 	<name value="Backup"/>
 	<conduit-attribute name="description" value="Backs up changed databases"/>
 	<conduit-attribute name="default-synctype" value="custom"/>
--- origsrc/gnome-pilot-2.32.0/conduits/evolution-data-server/e-address.conduit.in	2010-08-09 03:02:09.000000000 -0500
+++ src/gnome-pilot-2.32.0/conduits/evolution-data-server/e-address.conduit.in	2010-12-14 19:25:53.263730900 -0600
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="e_address_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/libeaddress_conduit.so"/>
+	<conduit id="e_address_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/cygeaddress_conduit.dll"/>
 	<name value="EAddress"/>
 	<conduit-attribute name="description" value="Synchronizes Addressbook with Evolution-Data-Server"/>
 	<conduit-attribute name="default-synctype" value="synchronize"/>
--- origsrc/gnome-pilot-2.32.0/conduits/evolution-data-server/e-calendar.conduit.in	2010-08-09 03:02:09.000000000 -0500
+++ src/gnome-pilot-2.32.0/conduits/evolution-data-server/e-calendar.conduit.in	2010-12-14 19:26:05.104151600 -0600
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="e_calendar_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/libecalendar_conduit.so"/>
+	<conduit id="e_calendar_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/cygecalendar_conduit.dll"/>
 	<name value="ECalendar"/>
 	<conduit-attribute name="description" value="Synchronizes Calendar with Evolution-Data-Server"/>
 	<conduit-attribute name="default-synctype" value="synchronize"/>
--- origsrc/gnome-pilot-2.32.0/conduits/evolution-data-server/e-memo.conduit.in	2010-08-09 03:02:09.000000000 -0500
+++ src/gnome-pilot-2.32.0/conduits/evolution-data-server/e-memo.conduit.in	2010-12-14 19:26:12.685764800 -0600
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="e_memo_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/libememo_conduit.so"/>
+	<conduit id="e_memo_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/cygememo_conduit.dll"/>
 	<name value="EMemos"/>
 	<conduit-attribute name="description" value="Synchronizes Memo List with Evolution-Data-Server"/>
 	<conduit-attribute name="default-synctype" value="synchronize"/>
--- origsrc/gnome-pilot-2.32.0/conduits/evolution-data-server/e-todo.conduit.in	2010-08-09 03:02:09.000000000 -0500
+++ src/gnome-pilot-2.32.0/conduits/evolution-data-server/e-todo.conduit.in	2010-12-14 19:26:20.142577800 -0600
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="e_todo_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/libetodo_conduit.so"/>
+	<conduit id="e_todo_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/cygetodo_conduit.dll"/>
 	<name value="EToDo"/>
 	<conduit-attribute name="description" value="Synchronizes ToDo List with Evolution-Data-Server"/>
 	<conduit-attribute name="default-synctype" value="synchronize"/>
--- origsrc/gnome-pilot-2.32.0/conduits/file/file.conduit.in	2009-08-06 13:59:19.000000000 -0500
+++ src/gnome-pilot-2.32.0/conduits/file/file.conduit.in	2010-12-14 19:25:30.394091000 -0600
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="gpfile1" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/libfile_conduit.so"/>
+	<conduit id="gpfile1" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/cygfile_conduit.dll"/>
 	<name value="File"/>
 	<conduit-attribute name="description" value="Installs files on the PDA"/>
 	<conduit-attribute name="default-synctype" value="custom"/>
--- origsrc/gnome-pilot-2.32.0/conduits/test/test.conduit.in	2009-08-06 13:59:19.000000000 -0500
+++ src/gnome-pilot-2.32.0/conduits/test/test.conduit.in	2010-12-14 19:25:30.394091000 -0600
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="test_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/libtest_conduit.so"/>
+	<conduit id="test_conduit" type="shlib" location="@LIBDIR@/gnome-pilot/conduits/cygtest_conduit.dll"/>
 	<name value="Test"/>
 	<conduit-attribute name="description" value="Test conduit"/>
      	<conduit-attribute name="default-synctype" value="synchronize"/>
